# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fetch/FetchTypes.ipdlh
# Commit: b9d73f961292
# Full Hash: b9d73f9612929dda88ca7e68d8d2e04de2aa2a86
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2023-01-20 21:21:03
# Regressor Bug: 1351231
# File Overlap Count: 2
# Description:
#   Bug 1351231 - FetchService integration for PFetch. r=dom-worker-reviewers,jesup
#   
#   Depends on D142436
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D142437
# ==============================================================================

diff -r 989cbb08b87e -r b9d73f961292 dom/fetch/FetchTypes.ipdlh
--- a/dom/fetch/FetchTypes.ipdlh	Fri Jan 20 09:09:17 2023 +0000
+++ b/dom/fetch/FetchTypes.ipdlh	Fri Jan 20 09:09:17 2023 +0000
@@ -35,8 +35,9 @@
   nsID uuid;
 };
 
-struct ParentToChildStream {
-  RemoteLazyInputStream stream;
+union ParentToChildStream {
+  RemoteLazyInputStream;
+  IPCStream;
 };
 
 struct ChildToParentStream {
@@ -86,6 +87,9 @@
   nsCString alternativeDataType;
   nsITransportSecurityInfo securityInfo;
   PrincipalInfo? principalInfo;
+  nsCString bodyBlobURISpec;
+  nsString bodyLocalPath;
+  RequestCredentials credentialsMode;
 };
 
 struct ParentToParentInternalResponse {